marketplace_builder/views/pages/api-reference/graphql/graphql-glossary.liquid

```diff
@@ -0,0 +1,189 @@
+
+---
+converter: markdown
+metadata:
+  title: GraphQL Glossary
+slug: api-reference/graphql/graphql-glossary
+searchable: true
+---
+
+This is a glossary of GraphQL related terms with examples from Platform OS implementations. 
+
+## Argument
+A set of key-value pairs attached to a specific field. Arguments can be literal values or variables.
+
+```graphql
+{
+  human(id: "200") {
```

Comment:
@diana-lakatos I think we should use existing endpoints from our system in examples
```graphql
user(id: 200) {
  email
  bio: property(name: "bio")
}
```